### DISCUSSION TITLE
Observer directory and execute scripts.

### DISCUSSION THREAD

**Discussion by julioflima** [dbeaver/dbeaver#14483] [Ideas]
Enlightened about a little project in Linux that execute all scripts from a dir, I thought that was an feature that should be included in amazing tool like Dbeaver. 

The feature is something like this:
- Should be just a modal to add multiple dirs that observe changes and execute on change.
- Could be interesting if was a icon button to enable and disable all observation.

> Sugested name: Gather

**Comment by kseniiaguzeeva** [comment]
Is it possible to give more details how it should look like?
